vars/deploy-on-openstack.yml

```diff
@@ -0,0 +1,31 @@
+---
+deploy_on_openstack_cloudname: "{{ lookup('env','OS_CLOUD') | default('engcloud-cloud-ci', true) }}"
+deploy_on_openstack_prefix: "{{ lookup('env','PREFIX') }}" #PREFIX is set at all times (ensured in script library)
```

Comment:
Sorry, but what exactly ensures PREFIX is set at all times? 
(And if there's no ensuring, should we have some default value if PREFIX is not set? Or should we fail if it is empty?)

Reply:
All the "actions" that requires openstack commands have a pre-flight check that enforce a failure should the env var be empty. I didn't change this.

Reply:
I like your idea of having pre-flight checks using ansible though.

Reply:
Ah, so we have a check for the final values, not for PREFIX itself